shard.yml

```
@@ -1,21 +1,22 @@
 name: ameba
-version: 1.5.0
+version: 1.6.0
 
 authors:
   - Vitalii Elenhaupt <velenhaupt@gmail.com>
+  - Sijawusz Pur Rahnama <sija@sija.pl>
 
 targets:
   ameba:
     main: src/cli.cr
 
 scripts:
-  # TODO: remove pre-compiled executable in future releases
   postinstall: shards build -Dpreview_mt
 
+# TODO: remove pre-compiled executable in future releases
 executables:
   - ameba
   - ameba.cr
 
-crystal: "~> 1.9.0"
+crystal: ~> 1.9
```

since we drop the compatibility with crystal 1.9 below, shouldn't we bump it to 1.10?

---

We do? If yes, I missed that.

---

We remove the code that ensures BC for crystal <= 1.9

https://github.com/crystal-ameba/ameba/pull/425/files#diff-3c764e731960f30a76c6f13b0387d5494855d0f52c8fa1a66df17637e3450680L79-L106

---

Which is fine, however, I expected to bump the version here

```suggestion
crystal: ~> 1.10
```

---

Done 🧹